packages/lab/stories/checkbox/checkbox.stories.tsx

```diff
@@ -85,6 +46,45 @@ export const Indeterminate: ComponentStory<typeof Checkbox> = () => {
   );
 };
 
+export const Error: ComponentStory<typeof Checkbox> = () => {
+  return (
+    <CheckboxGroup>
+      <Checkbox error checked={false} label="disabled checkbox" />
+      <Checkbox error indeterminate label="disabled indeterminate checkbox" />
+      <Checkbox error checked label="disabled checked checkbox" />
+    </CheckboxGroup>
+  );
+};
+
+export const Disabled: ComponentStory<typeof Checkbox> = () => {
+  return (
+    <CheckboxGroup>
+      <Checkbox disabled label="disabled checkbox" />
+      <Checkbox
+        disabled
+        indeterminate
+        label="disabled indeterminate checkbox"
+      />
```

Comment:
styling on disabled indeterminate seems to be wrong. shouldn't have hover styling